lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1189,6 +1192,13 @@ instance NFData StartTime
 newtype ProtocolMagic = ProtocolMagic Int32
     deriving (Generic, Show)
 
+{-------------------------------------------------------------------------------
+                                Accounts
+-------------------------------------------------------------------------------}
+
+newtype ChimericAccount = ChimericAccount ByteString
```

Comment:
What is this? A doc-comment 🙏 ?

Is it an account address, or an account PK? And it's raw binary I guess?

Reply:
added doc